Xamarin.PropertyEditing.Mac/Controls/Custom/BasePopOverViewModelControl.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using CoreGraphics;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class BasePopOverViewModelControl : BasePopOverControl
+	{
+		internal IEnumerable<object> Targets => this.viewModel.Editors.Select (ed => ed.Target);
+
+		internal IPropertyInfo Property => this.viewModel.Property;
+
+		private PropertyViewModel viewModel;
```

Comment:
You don't need the field here, you can just as well do `internal PropertyViewModel ViewModel {get;}`.

Reply:
Yeah it is a little overkill. I was thinking it would be needed down the line, but no point pre-empting that.

Reply:
Actually thinking about it, these BasePopOverControl changes should go in a separate PR as they don't directly affect PropertyReset, but the PRs that are coming based off PropertyReset. I'll create a new stand alone PR for these changes.